apps/ethos/repl-docmosis-service/demo.yaml

```diff
@@ -9,6 +9,6 @@ spec:
   releaseName: repl-docmosis-service
   values:
     java:
-      image: hmctspublic.azurecr.io/ethos/repl-docmosis-backend:prod-64eb548-20240725074610 #{"$imagepolicy": "flux-system:repl-docmosis-service"}
+      image: hmctspublic.azurecr.io/ethos/repl-docmosis-backend:prod-64eb548-20240725074610 #{"$imagepolicy": "flux-system:demo-repl-docmosis-service"}
```

Comment:
do you need to replace prod-blah-blah with pr-2401-blah-blah too?

Reply:
It will be automatically replaced when the image policy kicks in